apps/laboratory/tests/basic-tests.spec.ts

```diff
@@ -2,7 +2,7 @@ import { testM, expect } from './shared/fixtures/w3m-fixture'
 
 testM.describe('Modal only tests', () => {
   testM('Should be able to open modal', async ({ modalPage }) => {
-    await modalPage.page.getByText('Connect Wallet').click()
-    await expect(modalPage.page.getByText('All Wallets')).toBeVisible()
+    await modalPage.page.getByTestId('connect-button').click()
```

Comment:
Can we use a global constant for this so we have static type checking? Then the test ID itself can be pretty arbitrary, and it's easier to rename everything as things evolve.

Reply:
Made myself a follow up ticket for it https://github.com/WalletConnect/web3modal/issues/1582